src/Shared/Components/SelectPicker/type.ts

```diff
@@ -100,6 +100,8 @@ export type SelectPickerProps<OptionValue = number | string, IsMulti extends boo
     | 'autoFocus'
     | 'onBlur'
     | 'onKeyDown'
+    | 'onInputChange'
+    | 'inputValue'
```

Comment:
let's discuss once why this was required.

Reply:
Required to retaine searched value. We can diss in case any doubt

Reply:
The only issue i have with this is that this prop is open for all the use cases